.github/scripts/config-mingw.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-source `dirname $0`/config.sh
```

Comment:
why it should be changed?

Reply:
After the scripts moving, the relative path using `dirname` does not work when `config.sh` is imported from `config-mingw.sh` from scripts in different locations. For this reason, the top script needs to import both `config.sh` and `config-mingw.sh` directly. When `config.sh` is imported from the top script, it's not needed to be imported here.

Reply:
please try, it should work fine
source `dirname ${BASH_SOURCE[0]}`/config.sh